vllm/model_executor/models/qwen3_vl.py

```diff
@@ -741,20 +741,51 @@ def get_dummy_mm_data(
     ) -> MultiModalDataDict:
         num_images = mm_counts.get("image", 0)
         num_videos = mm_counts.get("video", 0)
+        image_overrides = mm_options.get("image") if mm_options else None
+        video_overrides = mm_options.get("video") if mm_options else None
 
         target_width, target_height = (
             self.info.get_image_size_with_most_features())
         target_num_frames = self.info.get_num_frames_with_most_features(
             seq_len, mm_counts)
+
+        if video_overrides:
+            num_frames_override = video_overrides.num_frames
```

Comment:
Will there always be .num_frames or .width? To be safe we might want to assert video_overrides is VideoDummyOptions